02-peer-protocol.md

```diff
@@ -2377,6 +2378,52 @@ be trimmed at the updated feerate, this could overflow the configured
 `max_dust_htlc_exposure_msat`. Whether to close the channel preemptively
 or not is left as a matter of node policy.
 
+### Alternative Addresses: `alt_addr`
```

Comment:
Think this should be done in either a BOLT01 `init` msg or a BOLT07 `node_announcement` msg, and not in the channel state machine.